io/CMakeLists.txt

```diff
@@ -320,7 +333,7 @@ if(build)
 
     add_definitions(${VTK_DEFINES})
     PCL_ADD_LIBRARY("${LIB_NAME}" "${SUBSYS_NAME}" ${srcs} ${incs} ${compression_incs} ${impl_incs} ${OPENNI_INCLUDES} ${OPENNI2_INCLUDES})
-    target_include_directories(${LIB_NAME} PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include" ${VTK_INCLUDE_DIRECTORIES})
+    target_include_directories(${LIB_NAME} PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include" ${VTK_INCLUDE_DIRECTORIES} ${REALSENSE2_INCLUDE_DIRS})
```

Comment:
Why add <code>${REALSENSE2_INCLUDE_DIRS}</code> in this line?
(Why doesn't add it in <code>PCL_ADD_LIBRARY()</code> on the upper line?)

Reply:
I don't know details of the PCL_ADD_LIBRARY macro, but I believe it accepts a list of _files_ to include in the compilation, where this is adding an additional include _directory_.

Reply:
The call to `target_link_libraries(${LIB_NAME} ${REALSENSE2_LIBRARIES})` below will setup include directories automatically.

Reply:
@taketwo In this case, Which do you think is better?

Reply:
> The call to target_link_libraries(${LIB_NAME} ${REALSENSE2_LIBRARIES}) below will setup include directories automatically.

@taketwo -- that's a new trick to me -- do you have a link to some docs? 

I didn't see anything obvious in the latest cmake docs regarding this (except for _maybe_ on mac?):
- https://cmake.org/cmake/help/v3.10/command/target_link_libraries.html?highlight=target_link_libraries#target-link-libraries

Reply:
This is an old feature of CMake (has been around since version 2). [Here](https://schneide.wordpress.com/2016/04/08/modern-cmake-with-target_link_libraries/) is short write-up about it.

Reply:
>  In this case, Which do you think is better?

It is necessary to link the library, so we have to have `target_link_libraries`. And as I wrote, this will set up includes automatically, so we can skip explicit manipulation of include directories.

Reply:
excellent - thanks for the tip, and a link to the article

built/tested locally -- fixed: 5130c5e554139657819e957175f3d439d8c88ff7